cluster-autoscaler/go.mod

```diff
@@ -25,6 +25,8 @@ require (
 	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a
 	golang.org/x/oauth2 v0.0.0-20210514164344-f6687ab2804c
 	google.golang.org/api v0.46.0
+	google.golang.org/grpc v1.38.0
```

Comment:
Kubernetes already uses grpc in a different version: https://github.com/kubernetes/kubernetes/blob/master/go.mod#L102, which makes it a transitive dependency of CA. In my experience is that different grpc versions don't always play well together. Right now the difference in version number is pretty small, but is there some way in which we can guarantee the same grpc version is used by CA and kubernetes going forward?

Reply:
I can use the same version that that kubernetes repo is using pretty easily, but do you have suggestions on how to keep it synced? I did a cursory search across other grpc uses in the k8s repo and found many repos seem to use many different versions:

1. https://github.com/kubernetes/node-problem-detector/blob/fc0edbd222e16b775ec58a253366a41822ecb3d5/vendor/modules.txt#L348
2. https://github.com/kubernetes/ingress-gce/blob/b1b62298cfc7adb08154402f2b26607d1a94d65e/vendor/modules.txt#L191
3. https://github.com/kubernetes/kops/blob/e40fd308b26676ef3f054e702640d32fecaa9271/vendor/modules.txt#L946
4. https://github.com/kubernetes/cloud-provider-openstack/blob/197604cb32c5d1850aeb8c903b7ea280f9bc0e1e/go.mod#L32
5. etc...